katex.js

```diff
@@ -14,11 +15,18 @@ import buildTree from "./src/buildTree";
 import parseTree from "./src/parseTree";
 import utils from "./src/utils";
 
+import type {SettingsOptions} from "./src/Settings";
+import type ParseNode from "./src/ParseNode";
+
 /**
  * Parse and build an expression, and place that expression in the DOM node
  * given.
  */
-let render = function(expression, baseNode, options) {
+let render = function(
+    expression: string,
+    baseNode: Node,
```

Comment:
Should this be `ParseNode`?

Reply:
This is an HTML [`Node`](https://developer.mozilla.org/en-US/docs/Web/API/Node). Should it perhaps be an [`HTMLElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) instead?

Reply:
I didn't read the surrounding code.  `Node` is fine since that's what `applyNode` is defined on.